projects/mercury/src/common/IconAPI.js

```diff
@@ -0,0 +1,26 @@
+import axios from 'axios';
+
+export type ObjectWithIconPath = {
+    iconPath: string
+};
+
+export type ObjectWithIconObjectUrl = ObjectWithIconPath & {
+    icon: string
+};
+
+export const getSvgIcon = (object: ObjectWithIconPath): Promise<ObjectWithIconObjectUrl> => {
+    if (object.iconPath === null) {
+        return Promise.resolve(object);
+    }
+    return axios.get(object.iconPath, {responseType: 'blob'}).then(response => {
```

Comment:
do I understand correctly, we don't cache now the icons? They are pretty static, something to make sense to cache in browser. What do you think?

Reply:
Yes, you're right, I had a plan to add caching, but then forgot about it! Thanks, I'll add it!

Reply:
It is added now